### PR TITLE
feat(cb2-3394): added wms view for allocated and atf cancelled events

### PR DESCRIPTION
**Pull request by tom-bjss**

### CODE REVIEW

File: views.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW daily_booking_alterations as
```
Comment (reisedawson-bjss) [inline views.sql:1]: Missed capitalisation of "as"

Comment (reisedawson-bjss) [inline views.sql:1]: Can we get a brief comment at the top about what this is? Even if a one-liner.

File: views.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    st.STAFF_ID,
+    se.STATUS,
+    se.EVENT_DATE,
+    se.EVENT_START,
+    se.EVENT_END
+FROM EDH_WMS_STAGING.NGT_SITE_EVENTS se
+JOIN EDH_WMS_STAGING.NGT_SITE s
+ON se.SITE_ID = s.SITE_ID
+JOIN EDH_WMS_STAGING.NGT_STAFF st
+ON se.STAFF_ID = st.STAFF_ID
```
Comment (reisedawson-bjss) [inline views.sql:13]: would typically indent lines 11 and 13 if not keeping on the same line as the table name

**Review by reisedawson-bjss — APPROVED** [review]

**Comment by bristol-mike-jones** [comment]
Not vital - but can you move the script into a SQL older like the nop

**Pull request merged**